File: src/com/falconraptor/timegame/resources/Resources.java
package com.falconraptor.timegame.resources;

import com.falconraptor.timegame.resources.fuel.*;
import com.falconraptor.timegame.resources.metals.*;
import com.falconraptor.timegame.resources.money.*;
import com.falconraptor.timegame.resources.ores.*;
import com.falconraptor.timegame.resources.utilities.*;

public class Resources {
	public Fuels fuels = new Fuels();
	public Metals metals = new Metals();
	public Money money = new Money();
	public Utilities utilities = new Utilities();
	public Crystals crystals = new Crystals("0");
	public Diamonds diamonds = new Diamonds("0");
	public Stone stone = new Stone("0");
	public Wood wood = new Wood("0");
	public Ores ores = new Ores();
}

File: src/com/falconraptor/timegame/resources/Resources.scala
package com.falconraptor.timegame.resources

import com.falconraptor.timegame.resources.fuel._
import com.falconraptor.timegame.resources.metals._
import com.falconraptor.timegame.resources.money._
import com.falconraptor.timegame.resources.ores._
import com.falconraptor.timegame.resources.utilities._

class Resources {
  var fuels: Fuels = new Fuels
  var metals: Metals = new Metals
  var money: Money = new Money
  var utilities: Utilities = new Utilities
  var crystals: Crystals = new Crystals("0")
  var diamonds: Diamonds = new Diamonds("0")
  var stone: Stone = new Stone("0")
  var wood: Wood = new Wood("0")
  var ores: Ores = new Ores
}